examples/spi/README.rst

```diff
@@ -33,9 +33,10 @@ Prerequisites
    aspects which are not specific to this application note are linked to in
    the references appendix.
 
- * For a description of XMOS related terms found in this document
+.. TODO: This link is not reachable. Remove or update it.
```

Comment:
if these are comments, I believe it is better to remove them or leave a comment about what needs to be added.

Reply:
removed